src/libcrun/criu.c

```diff
@@ -460,6 +466,42 @@ checkpoint_cgroup_v1_mount (runtime_spec_schema_config_schema *def, libcrun_erro
   return 0;
 }
 
+static int
+handle_criu_config_file (libcrun_container_t *container, libcrun_error_t *err)
+{
+  int ret;
+  const char *criu_config_annotation;
+  const char *config_file = CRIU_RUNC_CONFIG_FILE;
+
+  criu_config_annotation = find_annotation (container, "org.criu.config");
```

Comment:
we need to make sure this annotation prefix `org.criu.*` is added to the list of `potentiallyUnsafeConfigAnnotations` as it is visible in `crun features`

Reply:
Thanks! I've added this annotation prefix to `potentiallyUnsafeConfigAnnotations` in `src/libcrun/container.c` and `tests/test_oci_features.py`.